ikarus/finiteelements/feresulttypes.hh

```diff
@@ -135,6 +135,8 @@ namespace ResultTypes {
   REGISTER_SIMPLE_RESULTTYPE(BField, worldDim, 1);
   REGISTER_SIMPLE_RESULTTYPE(HField, worldDim, 1);
 
+  REGISTER_SIMPLE_RESULTTYPE(axialForce, gridDim, gridDim);
```

Comment:
The name is restricted to 1dimensional grid but you wrote gridDim. e.g. `axialForce` and `gridDim==3` does not make sense. Maybe explicitly write 1 or? @henrij22 does something speaks against this?

Reply:
Yes thats what I indicated